packages/apputils/src/instancetracker.ts

```diff
@@ -316,6 +316,10 @@ class InstanceTracker<T extends Widget> implements IInstanceTracker<T>, IDisposa
     return Promise.all(promises).then(([saved]) => {
       return Promise.all(saved.map(item => {
         const { id, value } = item;
+        if (!value) {
```

Comment:
Hm, I was thinking:

```
const args = value && (value as any).data;
if (!args) {
  return state.remove(item.id);
}
```

Reply:
Because `state.remove()` returns a `Promise<void>` and also because `(value as any).data` could inn theory be `false` but that's a valid value whereas the check for *just* `!value` still might allow for a value that is, say, `[]` or `5` or '"some string"`, etc.

Reply:
Good point.